tests/unit/test_wheel.py

```diff
@@ -17,6 +17,8 @@
 from pip._internal.utils.misc import unpack_file
 from tests.lib import DATA_DIR, assert_paths_equal
 
+from pip._internal.commands.wheel import WheelCommand
```

Comment:
Regarding Linter failures - This line needs to move. As the patch printed in the CI shows, this import is expected to be grouped with other imports from pip, sorted alphabetically.

You can install and run isort to make it do that or manually make the changes suggested by the diff in the CI.

$ pip install isort 
$ isort